Include libgen.h for POSIX basename as musl doesn provide GNU basename in string.h.

diff -rup src/wiphy.c.orig src/wiphy.c
--- a/src/wiphy.c
+++ b/src/wiphy.c
@@ -33,6 +33,7 @@
 #include <unistd.h>
 #include <string.h>
 #include <limits.h>
+#include <libgen.h>
 
 #include <ell/ell.h>
 
